package/MDAnalysis/analysis/align.py

```diff
@@ -239,9 +241,10 @@ def rotation_matrix(a, b, weights=None):
 
     See Also
     --------
-    :func:`rmsd` calculates the RMSD between *a* and *b*; for fitting a whole
-    trajectory it is more efficient to use :func:`rms_fit_trj`. A complete fit
-    of two structures can be done with :func:`alignto`. """
+    MDAnalysis.analysis.rms.rmsd: Calculates the RMSD between *a* and *b*.
```

Comment:
Are the functions/classes automatically hyperlinked or does one need markup, e.g. :func:\`alignto\`?